gprofiler/profilers/java.py

```diff
@@ -784,28 +769,34 @@ def start(self) -> None:
             self._saved_mlock = read_perf_event_mlock_kb()
             write_perf_event_mlock_kb(self._new_perf_event_mlock_kb)
 
+        # needs to run in init net NS
+        run_in_ns(["net"], lambda: proc_events.register_exit_callback(self._proc_exit_callback), 1)
```

Comment:
As we discussed verbally, we should wrap it in a `try...except` or at least leave a `TODO` as a reminder

Reply:
Done